### DISCUSSION TITLE
how to send and receive data via hub device.

### DISCUSSION THREAD

**Discussion by Lafengmao** [hathach/tinyusb#2186] [Q&A]
Our board works as  a usb host. When a usb device insert it, the    tuh_mount_cb    function is called.
I know it means the device has been mounted. 
My question is : are there any existing api to  send data to the device or read data from the usb device? just like the CDC api below:
tuh_cdc_read and tuh_cdc_write.
Thanks.

**Comment by hathach** [comment]
Which type of device is plugged in ? You will need the corresponding API to interact with. Try to run and read examples for how to

**Reply by Lafengmao** [reply]
Hi, 
it is a  usb-acm device, a 4G LTE module. Do we have existing APi to interact with this kind of device?
thanks very much.
